packages/datagrid/src/components/DataGrid/DataGrid.scss

```diff
@@ -67,6 +67,10 @@ $td-datagrid-skeleton-height: 9.6rem !default;
 		font-family: inherit;
 	}
 
+	:global(.ag-horizontal-left-spacer) {
+		overflow-x: hidden;
```

Comment:
Why is not `auto`?

Reply:
it is only the pin column, the pin column never overflows